consensus_pbft.go

```diff
@@ -792,6 +793,31 @@ func (p *Pbft) PushMessage(msg *MessageReq) {
 		return
 	}
 
+	// wanted to do Commit Validation in parallel
+	// for that reason, validating logic was moved from runValidateState
+	if msg.Type == MessageReq_Commit {
+		p.msgQueue.pushPendingCommitMessage(msg)
+		if p.state.view == nil {
+			return
+		}
+		p.msgQueue.processPendingCommitMessages(p.state.view.Copy(), p.verifyCommitMessage)
```

Comment:
I would imagine that the `msgQueue` should figure out if there is any new value in `pendingCommitQueue` and inject into the verification workflow if necessary.